geode-apis-compatible-with-redis/src/test/java/org/apache/geode/redis/internal/SupportedCommandsJUnitTest.java

```diff
@@ -91,6 +91,7 @@
       "ZINCRBY",
       "ZCARD",
       "ZCOUNT",
+      "ZPOPMIN",
```

Comment:
Could this be moved to below ZPOPMAX to maintain alphabetical ordering?

Reply:
Done